opossum-tool/OpossumUI
# feat: make checkbox labels clickable

### Summary of changes

- improve and simplify Checkbox component to include the checkbox label as clickable element
- moved "Source" to new line because on smaller screens there never was enough space to display it properly

### Context and reason for change

Closes #2349 .

### How can the changes be tested

Check checkboxes and their labels in the following components:
- auditing sub-panel ("1st party" etc.)
- edit attribution popup (report view)
- locator popup
- "needs review" button in attribution details
